### PR TITLE
Clean up colour filters

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

File: src/drawing/NewDrawing.cpp [hunk 1]
```diff
@@ -210,6 +210,15 @@ extern "C"
         }
     }
 
+    void gfx_filter_rect(rct_drawpixelinfo * dpi, int left, int top, int right, int bottom, filter_palette palette)
+    {
+        if (_drawingEngine != nullptr)
+        {
+            IDrawingContext * dc = _drawingEngine->GetDrawingContext(dpi);
+            dc->FilterRect(palette, left, top, right, bottom);
+		}
```
Comment (IntelOrca) [inline src/drawing/NewDrawing.cpp]: Brace not aligned

File: src/drawing/drawing.h [hunk 2]
```diff
@@ -87,6 +88,46 @@ enum {
...
+	PALETTE_TRANSLUCENT_LIGHT_ORANGE = 94,
+	PALETTE_TRANSLUCENT_TEAL = 97,
+	PALETTE_TRANSLUCENT_BRIGHT_PINK = 100,
+	PALETTE_TRANSLUCENT_DARK_BROWN = 103,
+	PALETTE_TRANSLUCENT_LIGHT_PINK = 106,
+	PALETTE_TRANSLUCENT_WHITE = 109,
+
+	PALETTE_132 = 112 + COLOUR_LIGHT_ORANGE,
+	PALETTE_139 = 112 + COLOUR_SATURATED_RED,
+} filter_palette;
```
Comment (IntelOrca) [inline src/drawing/drawing.h]: enum names should be upper case, so `FILTER_PALETTE`

**Comment by marijnvdwerf** [comment]
@IntelOrca Care to review? I think I've got all `fill_rect` calls with `0x20000000` set.

**Comment by marijnvdwerf** [comment]
What about the code duplication in the drawing engines?

**Comment by IntelOrca** [comment]
Its not that similar, and its the same for all the other drawing functions.

**Pull request merged**